File: src/main/java/dev/zprestige/fire/module/misc/nointerpolation/TickListener.java
package dev.zprestige.fire.module.misc.nointerpolation;

import dev.zprestige.fire.newbus.EventListener;
import dev.zprestige.fire.newbus.events.TickEvent;

public class TickListener extends EventListener<TickEvent, NoInterpolation> {

    public TickListener(final NoInterpolation noInterpolation){
        super(TickEvent.class, noInterpolation);
    }

    @Override
    public void invoke(final Object object){
        module.i++;
    }
}

File: src/main/java/dev/zprestige/fire/module/misc/nointerpolation/TickListener.kt
package dev.zprestige.fire.module.misc.nointerpolation

import dev.zprestige.fire.event.bus.EventListener
import dev.zprestige.fire.event.impl.TickEvent

class TickListener(noInterpolation: NoInterpolation) : EventListener<TickEvent, NoInterpolation>(TickEvent::class.java, noInterpolation) {

    override fun invoke(e: Any) {
        module.i++
    }
}
